gsy_framework/sim_results/electric_blue/aggregators.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from typing import Dict
+
+
+class BaseAggregator(ABC):
+    """Base accumulator class."""
+    @abstractmethod
+    def __call__(self, collected_raw_data: Dict):
+        """Calculate and return accumulated results."""
+
+
+class EBEnergyProfileAggregator(BaseAggregator):
+    """Aggregator class for EB energy trade profile."""
+
+    DEFAULT_STATS = {
+        # open, still unmatched bids/offers
+        "open_buy_orders": 0.0,
+        "open_sell_orders": 0.0,
+        # matched bids/offers
+        "matched_buy_orders": 0.0,
+        "matched_sell_orders": 0.0,
+        # all
+        "all_buy_orders": 0.0,
+        "all_sell_orders": 0.0,
+    }
+
+    def __call__(self, collected_raw_data: Dict):
+        current_result = defaultdict(lambda: dict(self.DEFAULT_STATS))
+
+        for trade in collected_raw_data["trades"]:
+            seller_id = trade["seller_id"]
+            buyer_id = trade["buyer_id"]
+            energy = trade["energy"]
+            # SELLER
+            current_result[seller_id]["matched_sell_orders"] += energy
```

Comment:
In order to be explicit, I would append _kWh to all these dict keys to denote that this is energy:

```suggestion
            current_result[seller_id]["matched_sell_orders_kWh"] += energy
```

Reply:
Will fix that soon